.gitpod.Dockerfile

```
@@ -2,4 +2,18 @@ FROM gitpod/workspace-full
 
 USER gitpod
 
-RUN brew install R
+# Install R
+RUN sudo apt-get update \
+    && sudo apt-get install -y \
+        build-essential \
+        gfortran \
+        perl \
+        wget \
+        m4 \
+        cmake \
+        pkg-config \
+        r-base \
+    && sudo rm -rf /var/lib/apt/lists/*
```

I think, this can be easily resolved with the following:

```suggestion
RUN sudo apt install r-base --yes
```